### PR TITLE
mojave-gtk-theme: 2019-09-09 -> 2019-12-12

### PR DESCRIPTION
**Pull request by romildo**
###### Motivation for this change

- Update to version [2019-12-12](https://github.com/vinceliuice/Mojave-gtk-theme/releases/tag/2019-12-12)
- Add wallpapers

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/mojave/default.nix [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
-  };
+  srcs = [
+    (fetchFromGitHub {
+      owner = "vinceliuice";
+      repo = pname;
+      rev = version;
+      sha256 = "0d5m9gh97db01ygqlp2sv9v1m183d9fgid9n9wms9r5rrrw6bs8m";
+    })
+    (fetchurl {
+      url = "https://raw.githubusercontent.com/vinceliuice/Mojave-gtk-theme/images/macOS_Mojave_Wallpapers.tar.xz";
```
Comment (c0bw3b) [inline pkgs/data/themes/mojave/default.nix]: Is that URL stable enough over time?

Comment (romildo) [reply pkgs/data/themes/mojave/default.nix]: I think so. It has never changed in the git repository.

Screenshot from 2019-12-13 12-11-52

Comment (worldofpeace) [reply pkgs/data/themes/mojave/default.nix]: This url will be stable https://github.com/vinceliuice/Mojave-gtk-theme/raw/11741a99d96953daf9c27e44c94ae50a7247c0ed/macOS_Mojave_Wallpapers.tar.xz

Comment (romildo) [reply pkgs/data/themes/mojave/default.nix]: > This url will be stable https://github.com/vinceliuice/Mojave-gtk-theme/raw/11741a99d96953daf9c27e44c94ae50a7247c0ed/macOS_Mojave_Wallpapers.tar.xz

Changed to this URL.

**Review by c0bw3b — APPROVED** [review]

**Pull request merged**